vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,16 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
+      next if h[:container_node_name].nil?
+      node = ContainerNode.first(
+        :conditions => {:ems_id => ems.id, :name => h[:container_node_name]})
+      h[:container_node_id] = node.nil? ? nil : node.id
```

Comment:
Yeah, this is not right...the parser should be responsible for relating all of this information, and then the save_inventory naturally does the linkage.  Let's discuss separately.